packages/marathon/buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "requires": ["java"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.6.564-3fa693b32/marathon-1.6.564-3fa693b32.tgz",
-    "sha1": "d87fc13bff1f560e76f53b2205d9583566fddb8e"
+    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.6.567-2d8b3e438/marathon-1.6.567-2d8b3e438.tgz",
```

Comment:
is there a reason to prefer this over the usual `https://downloads.mesosphere.com/marathon/builds/1.6.567-2d8b3e438/marathon-1.6.567-2d8b3e438.tgz` ?

Reply:
@takirala the `downloads.mesosphere.com` is fronted by a CDN... if there is a need to update the artifact it is painful to fix... we moved to a standard of using `https` and using the direct `s3.amazonaws.com` long ago.   If we only build once successfully the CDN doesn't help anyway.